crt_portal/cts_forms/tests/integration-authed/report_detail.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from auth import login_as_superuser
+
+
+@pytest.mark.only_browser("chromium")
+def test_click_back_to_all(page):
+    """Opens report detail page and goes back to reports page"""
+
+    login_as_superuser(page)
+
+    page.goto("/form/view")
+
+    with page.expect_navigation():
+        page.evaluate("document.querySelector('.td-link').click()")
+
+    assert page.is_visible("#contact-info")
+
+    with page.expect_navigation():
+        page.evaluate("document.getElementById('backtoall').click()")
```

Comment:
I've found that in most cases, you can use the `click()` method on locator directly.

Also, to avoid assigning ids to things, you can use locator().filter() - for example:

```
    page.locator('.crt-landing--example_option').filter(has_text="Victim of a hate crime").click()

    assert page.locator('p').filter(has_text="Attacks, threats of violence, or destruction of property at place of worship").is_visible()
    assert page.locator('a').filter(has_text="Get help for hate crimes").is_visible()
```